IeeeVisUploader/IeeeVisUploaderWebApp/config/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "BunnyStorageZoneName": "",
+  "BunnyCdnRootUrl": "",
+  "BunnyAccessKey": "",
+  "BunnyBasePath": "",
+  "BunnyTokenKey": "",
+  "BunnyUserApiKey": "",
+  "AuthSignaturePrivateKey": "123456",
+  "FfprobePath": "/usr/local/bin/ffprobe",
+  "FfmpegPath": "/usr/local/bin/ffmpeg"
```

Comment:
This should be common for linux and mac machines, so just make them the default. We can add a note in the README for updating these for other types of systems.